### PR TITLE
add solution

### PR DESCRIPTION
**Pull request by Anastasiia-Khudych**
[Demo](https://anastasiia-khudych.github.io/react_people-table-basics/#/)

### CODE REVIEW

File: src/components/NavBar/NavBar.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+          className={({ isActive }) => classNames(
+            'navbar-item ',
+            {
+              'has-background-grey-lighter': isActive,
+            },
+          )}
+          to="/people"
+        >
+          People
+        </NavLink>
```
Comment (andrii-andriushchenko) [inline src/components/NavBar/NavBar.tsx:36]: I would to create separate component for `NavLink` for avoid dublicate code) but up for you)

**Review by andrii-andriushchenko — APPROVED** [review]
well done 🚀

File: src/components/NavBar/NavBar.tsx (hunk 1 above)
Comment (denys-cheporniuk) [reply src/components/NavBar/NavBar.tsx:36]: agree, let's create a separate component for `NavLink`

File: src/components/PeoplePage/PeoplePage.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+import { PeopleTable } from '../PeopleTable';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [hasLoadingError, setHasLoadingError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const loadPeople = async () => {
```
Comment (denys-cheporniuk) [inline src/components/PeoplePage/PeoplePage.tsx:12]: useCallback

File: src/components/PeopleTable/PeopleTable.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = (
+  {
+    people,
+  },
+) => {
+  return (
```
Comment (denys-cheporniuk) [inline src/components/PeopleTable/PeopleTable.tsx:14]: you can avoid return here

**Review by denys-cheporniuk — CHANGES_REQUESTED** [review]

**Review by denys-cheporniuk — APPROVED** [review]

**Pull request closed without merging**